src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/RootHiddenExpansion.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics;
+using Microsoft.VisualStudio.Debugger.Evaluation;
+using Microsoft.VisualStudio.Debugger.Evaluation.ClrCompilation;
+
+namespace Microsoft.CodeAnalysis.ExpressionEvaluator
+{
+    internal sealed class RootHiddenExpansion : Expansion
+    {
+        internal static Expansion CreateExpansion(
+            MemberAndDeclarationInfo members,
+            DynamicFlagsMap dynamicFlagsMap)
+        {
+            return new RootHiddenExpansion(members, dynamicFlagsMap);
+        }
+
+        private readonly MemberAndDeclarationInfo _member;
+        private readonly DynamicFlagsMap _dynamicFlagsMap;
+
+        internal RootHiddenExpansion(MemberAndDeclarationInfo member, DynamicFlagsMap dynamicFlagsMap)
+        {
+            _member = member;
+            _dynamicFlagsMap = dynamicFlagsMap;
+        }
+
+        internal override void GetRows(
+            ResultProvider resultProvider,
+            ArrayBuilder<EvalResultDataItem> rows,
+            DkmInspectionContext inspectionContext,
+            EvalResultDataItem parent,
+            DkmClrValue value,
+            int startIndex,
+            int count,
+            bool visitAll,
+            ref int index)
+        {
+            var memberValue = value.GetMemberValue(_member, inspectionContext);
+            var isDynamicDebugViewEmptyException = memberValue.Type.GetLmrType().IsDynamicDebugViewEmptyException();
+            if (isDynamicDebugViewEmptyException || memberValue.IsError())
+            {
+                if (InRange(startIndex, count, index))
+                {
+                    if (isDynamicDebugViewEmptyException)
```

Comment:
Does EmptyException have an "Empty" property? If so, could this be handled by MemberExpansion of the Exception instance, as in "Results View"?